@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -229,6 +242,8 @@ export class PickHandler {
             return a.distance - b.distance;
         });
 
+        this.mapView.update();
```

Comment:
Why is it necessary to update the mapview after the object picking?

Reply:
It doesn't, that's part of some test code I left there unintentionally. good catch.